apps/modernization-api/src/main/java/gov/cdc/nbs/patient/search/PatientSearchCriteriaSortResolver.java

```diff
@@ -31,7 +31,7 @@ private SortOptions asOption(final Sort.Order sorting) {
       case "city" -> asSortOption(ADDRESS, "address.city.keyword", order);
       case "email" -> asSortOption("email", "email.emailAddress.keyword", order);
       case "firstNm" -> asSortOption("name", "name.firstNm.keyword", order);
-      case "id" -> asSortOption("id", order);
+      case "id" -> asSortOption("patient", order);
```

Comment:
the "Patient ID" that the frontend is sorting on the the "short id" which is not available in the search index.  The closest thing we have is the `local_id`

Reply:
ok then that's a front end change to use local_id which is working.  this will fix patient id sorting.